src/base/porting.h

```diff
@@ -354,7 +354,7 @@ check_is_array (const T & a)
 
 #if defined (WINDOWS)
 #define IS_ABS_PATH(p) IS_PATH_SEPARATOR((p)[0]) \
-	|| (isalpha((p)[0]) && (p)[1] == ':' && IS_PATH_SEPARATOR((p)[2]))
+	|| (isalpha((p)[0]) && (p)[1] == ':' && (IS_PATH_SEPARATOR((p)[2]) || (p)[2] == '/'))
```

Comment:
I guess you need to check that p[0] == '/' is also an absolute path.
And if it's a window, how about not using '/' directly, but adding a variable defined like this:
#define LINUX_PATH_SEPARATOR '/'

Reply:
Yes, you right, but I want to minimize impact of modification so conservative correction was made.
Basically, I agree upon your suggestion, I will take a look for the impact after modification.

Reply:
OK, after checking impact of modification, I want to you consider my suggestion.

Reply:
My intention was to take your comment and decide following cases below to absolute paths in the Windows:
1. /abc/tmp.log
2. c:/abc/tmp.log (extended by this PR)
3. c:\abc/tmp.log (or c:\abc\tmp.log)

I'll ask for your opinion again.

1. keep the current revision of this PR for IS_ABS_PATH
2. includes all three cases above for the Windows absolute path 

Of course, this modification will have a greater impact than the previous one, but it is necessary when considering the environment such as MinGW.

Reply:
@mhoh3963, @airnet73
I think we have to take **option 2** and pushed the codes for it. 
- If you agree with my opinion and accept the changes, just approve or leave a comment. 
- Or please leave comments, then I'll reflect your opinion including back to previous commit. 
- This push is just for your choice for the options. Thank you.